x-pack/test_serverless/api_integration/test_suites/observability/platform_security/authorization.ts

```diff
@@ -10,8 +10,8 @@ import { FtrProviderContext } from '../../../ftr_provider_context';
 export default function ({ getService }: FtrProviderContext) {
   const svlCommonApi = getService('svlCommonApi');
 
-  describe('security/authorization', function () {
-    describe('available features', () => {
+  describe.skip('security/authorization', function () {
```

Comment:
It looks like this suite only contains test regarding privilege authorization utilizing the `composedOf` feature. We no longer utilize `composedOf` for our serverless feature privileges. Can I get rid of this suite entirely or is there something else we should test here? @azasypkin

Reply:
The Observability project type still relies on the feature overrides/composedOf [defined in the base `serverless.yml`](https://github.com/elastic/kibana/blob/a2bd0a73c6d44c42b3b62c457d8a6956049c6571/config/serverless.yml#L15-L101), so it makes sense to still keep these tests in case something changes in the base config, since it will affect your solution and might not reflect how you expect your product to behave